airbyte-cron/src/main/java/io/airbyte/cron/selfhealing/Temporal.java

```diff
@@ -4,19 +4,27 @@
 
 package io.airbyte.cron.selfhealing;
 
+import io.airbyte.workers.temporal.TemporalClient;
 import io.micronaut.scheduling.annotation.Scheduled;
+import io.temporal.api.enums.v1.WorkflowExecutionStatus;
+import javax.inject.Inject;
 import javax.inject.Singleton;
 import lombok.extern.slf4j.Slf4j;
 
 @Singleton
 @Slf4j
 public class Temporal {
 
+  @Inject
+  private TemporalClient temporalClient;
+
```

Comment:
I'm curious if we have stance on private field injections vs constructor/setter injection. My preference would be to convert this to use constructor injection.  I have found that private field injections tend to make code more difficult to test.

Reply:
I don't have a strong preference but I agree that it will be easier to test.